helm/alfresco-process-infrastructure/values.yaml

```diff
@@ -119,7 +119,7 @@ alfresco-modeling-service:
       readOnly: true
   image:
     repository: quay.io/alfresco/alfresco-modeling-service
-    tag: 7.1.0-M7
+    tag: "develop"
```

Comment:
no need for quotes

Reply:
makes sense i got that from the history, changing it now